feature/system/logging/console_vty_file/tests/README.md

```diff
@@ -0,0 +1,81 @@
+# TR-6.2 Local logging destinations
```

Comment:
Is this added to the https://github.com/openconfig/featureprofiles/blob/main/testregistry.textproto#L650-L660?